specification.md

```diff
@@ -700,8 +746,9 @@ Furthermore, implementations must, to the extent possible, simulate missing
 states. For example, if the implementation polls a LRM queue infrequently 
 enough such that the active state of a job is skipped between two polling 
 rounds, the job would appear to have jumped from a `QUEUED` state to a 
-`COMPLETED` state. However, implementations can introduce a synthetic `ACTIVE` 
-state change.
+`COMPLETED` state. However, implementations can, for example, introduce a 
+synthetic `ACTIVE` state change.
+
```

Comment:
I would suggest to change `can` to `must` to be in sync with the requirement that `STAGE_IN` and `STAGE_OUT` transitions are generated even when no data are staged.

Reply:
I agree. My thought was that "must introduce a synthetic ACTIVE change" should not really apply if there is a real ACTIVE change. This should be reworded to clarify that *if the real one is missing*, a synthetic one must be introduced.

Reply:
And done